update_changelog.sh

```diff
@@ -5,7 +5,7 @@ which docker > /dev/null || (echoerr "Please ensure that docker is installed" &&
 
 cd -P -- "$(dirname -- "$0")" # switch to this dir
 
-CHANGELOG_FILE=CHANGELOG.md
+CHANGELOG_FILE=CHANGELOG_V1.md
```

Comment:
So this will need to be reverted in `main` after merging from `v1` (to update main `CHANGELOG.md`).

Reply:
Yes, but this in one-time conflict only.

Reply:
Yes, of course.